new-log-viewer/src/components/MenuBar/index.css

```diff
@@ -14,15 +14,28 @@
 .menu-bar-logo-container {
     display: flex;
     justify-content: center;
-    width: 48px;
+    min-width: 48px;
```

Comment:
This avoids the YScope logo container from being squeezed when the other items in the menu bar try to take space.